### ISSUE TITLE
Feedback area - Contact Form view shows "wrong" headline.

### ISSUE THREAD

**Issue by digitalwaveride** [Automattic/wp-calypso#33916]
#### Steps to reproduce
If you click on the "Feedback" menu time in the left sidebar, you get linked to /wp-admin/edit.php?post_type=feedback.
There you see by default the contact form area:

#### What I expected
The headline of that view should not say "Feedback" but instead "Contact Form", as the whole area is "Feedback" and the shown view is just one page of it -> "Contact Form".

#### Screenshot / Video
<img width="1439" alt="Screenshot 2019-06-12 at 15 45 08" src="https://user-images.githubusercontent.com/5120357/59392555-5305ed00-8d78-11e9-9ec7-129ca061895e.png">

#### Context / Source
Found during design meetup "Bottle Pickup" session

**Comment by allancole** [comment]
It seems we might be able to fix this by revising some of the labels for the Feedback custom-post-type in Jetpack here: https://github.com/Automattic/jetpack/blob/a52b3cd0330a8db3e54715dacc686ef70fa771d4/modules/contact-form/grunion-contact-form.php#L171

However, I don’t know if this will negatively impact other Jetpack modules/experiences where the term “Feedback” might actually be appropriate. 

May need a second opinion on how to best approach this. Cc: @MichaelArestad, @crunnells

**Comment by jeherve** [comment]
Closing as duplicate of https://github.com/Automattic/jetpack/pull/9674 and https://github.com/Automattic/jetpack/issues/9637 where there was some more discussion about it.

tl;dr: changing that name means changing it everywhere (in Calypso, in WordPress.com wp-admin, in multiple screens of any Jetpack site), and before to make any changes we should consider the impact it would have on all those screens.
